CONTRIBUTING.md

```diff
@@ -0,0 +1,63 @@
+# Contributing Guide
+
+## Development
+
+### Python Version
+
+Developing in this repository requires Python 3.10 or higher.
+
+### Set-up
+
+Clone the repo by running:
+
+```bash
+git clone git@github.com:coinbase/cdp-sdk-python.git
+```
+
+To install all dependencies, run:
+
+```bash
+make install-deps
+```
+
+### Linting
+
+To autocorrect all lint errors, run:
+
+```bash
+make lint-fix
+```
+
+To detect all lint errors, run:
+
+```bash
+make lint
+```
+
+### Testing
+
+Install the `pytest` and `pytest-cov` packages to run tests:
+
+```bash
+pip install pytest pytest-cov
+```
+
+To run all tests, run:
+
+```bash
+make test
+```
+
+### Generating Documentation
+
+To generate documentation from the Python docstrings, run:
+
+```bash
+make docs
+```
+
```

Comment:
this doesn't actually generate the documentation html files. It is a pre-build step and is primarily used in GHA.

`make local-docs` will build and open all docs for viewing!

Reply:
removed it